src/views/SearchSelfCareView.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react";
+import { Helmet } from "react-helmet";
+import SelfCareSearchBar from "../components/searchSelfCareView/SelfCareSearchBar";
+
+const SearchSelfCareView = () => {
+  const [searchResults, setSearchResults] = useState({});
```

Comment:
Where is this state being set?

Reply:
in SelfCareSearchBar.jsx

Reply:
Okay, we definitely need Redux to avoid this kind of passing of functionality as props.

Reply:
I am doing it in other PR. I think passing one level up or down is not a big deal but I also don't mind Redux)